gradio/blocks.py

```diff
@@ -1611,19 +1626,22 @@ def queue(
             demo.queue(max_size=20)
             demo.launch()
         """
-        if concurrency_count is None:
-            concurrency_count = 1 if utils.get_space() else 40
+        if "concurrency_count" in kwargs:
+            raise DeprecationWarning(
+                "concurrency_count has been deprecated. Set the concurrency_limit directly on event listeners e.g. btn.click(fn, ..., concurrency_limit=10) or gr.Interface(concurrency_limit=10). If necessary, the total number of workers can be configured via `max_threads` in launch()."
+            )
+        if len(kwargs):
```

Comment:
Won't this raise both a warning AND a ValueError if "concurrency_count" in kwargs?

Reply:
Nope, DeprecationWarning is actually an exception, not a warning

Reply:
But if you put `raise` before it, it will interrupt the program

Reply:
e.g. test with:

```py
a = "5"
raise DeprecationWarning
print(a)
```

I can fix it in my PR

Reply:
Oh never mind I misunderstood